tox.ini

```diff
@@ -16,21 +16,24 @@ isort==5.10.1
 mypy==0.931
 pylint==2.11.1
 pytest==7.0.0
+types-Jinja2==2.11.9
```

Comment:
I think this one is wrong. Our check and do format jobs run against latest pip rather than minimum supported, and latest pip for 3.6 installs 3.0.3.

According to https://pypi.org/project/types-Jinja2/ , 
> Note: The Jinja2 package includes type annotations or type stubs since version 3.0. Please uninstall the types-Jinja2 package if you use this or a newer version.